api/src/db_operations.py

```diff
@@ -115,7 +117,9 @@ def seed_db(config):
 if __name__ == '__main__':
     cmd = sys.argv[1]
     config = sys.argv[2]
-    if cmd == 'drop':
+    if config != 'dev' or config != 'test':
```

Comment:
Double check the logic here. It looks like it will print the error message no matter what ```config``` is.